src/Microsoft.AspNetCore.Identity/SignInManager.cs

```diff
@@ -433,7 +433,19 @@ public virtual Task ForgetTwoFactorClientAsync()
         /// <param name="isPersistent">Flag indicating whether the sign-in cookie should persist after the browser is closed.</param>
         /// <returns>The task object representing the asynchronous operation containing the <see name="SignInResult"/>
         /// for the sign-in attempt.</returns>
-        public virtual async Task<SignInResult> ExternalLoginSignInAsync(string loginProvider, string providerKey, bool isPersistent)
+        public virtual Task<SignInResult> ExternalLoginSignInAsync(string loginProvider, string providerKey, bool isPersistent) 
```

Comment:
Ah, well the original method was already virtual and should remain so, if they wish to override the method and change the behavior to call the new overload with byPassTwoFactor: true instead, that's their prerogative, its really up to the app what behavior they want for these flags.